application/org.openjdk.jmc.updatesite.ide/src/main/resources/update-site-instructions/index.html

```diff
@@ -76,7 +76,7 @@
                 <ol start="2">
                   <li>
                     <p>Fill in the following URL in the <b>Work with:</b> text field:</p>
-                    	<p><b>https://download.oracle.com/technology/products/missioncontrol/updatesites/openjdk/7.1.0/eclipse/</b></p>
+                    	<p><b>https://download.oracle.com/technology/products/missioncontrol/updatesites/openjdk/8.0.0/eclipse/</b></p>
```

Comment:
My terminal is showing new empty spaces on this line .. 
![2019-11-14-140606_1315x375_scrot](https://user-images.githubusercontent.com/10425301/68888327-bcec3f00-06e8-11ea-9ffb-d00cebba2b5f.png)

It seems to be because there's a tab just before the `<p>`, but up until that point it's single spaces.

Reply:
> Or will they be squashed when merged?

They should be automatically squashed.

Reply:
Funny, the following line also had a tab. Given the standard indentation of the project, they should all be tabs, but they got to be spaced for now.

We should add spotless...